src/Illuminate/Database/Query/Builder.php

```diff
@@ -2879,4 +2879,28 @@ public function __call($method, $parameters)
 
         static::throwBadMethodCallException($method);
     }
+
+    /**
+     * Print SQL query with bindings.
+     *
+     * @return string
+     */
+    public function toSqlBound()
+    {
+        $sql = $this->toSql();
+
+        $bindings = $this->getBindings();
+
+        if (count($bindings) == 0) {
+            return $sql;
+        }
+
+        array_walk($bindings, function ($value) use (&$sql) {
+            $value = is_string($value) ? var_export($value, true) : $value;
```

Comment:
Just `"'$value'"` instead of `var_export($value, true)`?

Reply:
Just in case :) 
Feel free to contribute to it.